station_data.py

```diff
@@ -1,16 +1,22 @@
 from datetime import datetime
-from urllib import urlencode
-from urllib2 import urlopen
+import requests
 from StringIO import StringIO
 from numpy import genfromtxt
 import ast
 
-station_ids = [ 2 ]
-
 base_url = "http://data.hisparc.nl/api/station/%d/"
 
-for id in station_ids:
-	url = urlopen(base_url % (id))
-	meta_data = ast.literal_eval(url.read())
-	print( type( meta_data ) )
-	print( meta_data )
+def extract_data(code):
+    r = requests.get(base_url%(code))
+    if r.status_code == requests.codes.ok:
+        return r.json()
+    else:
+        r.raise_for_status()
+
+def main():
+    station_ids = [2]
```

Comment:
The other pull requests (pr/1) gets the list of stations from another URL (http://data.hisparc.nl/api/stations/). Maybe combining both PR's could work too :-)